.vsts.pipelines/steps/setup-macos-native.yml

```diff
@@ -24,10 +24,7 @@ steps:
     brew upgrade openssl || true
     brew upgrade pkgconfig || true
     brew upgrade cmake || true
-
-    brew link --force icu4c
-    brew_openssl_paths=$(brew info openssl | cut -d ' ' -f 1 | grep '^/.*/openssl/')
-    # Use 'for' to get last/latest openssl install dir.
-    for openssl_path in $brew_openssl_paths; do :; done
-    setvariable OPENSSL_ROOT_DIR "$openssl_path"
+    ln -s /usr/local/opt/openssl/lib/pkgconfig/libcrypto.pc /usr/local/lib/pkgconfig/
+    ln -s /usr/local/opt/openssl/lib/pkgconfig/libssl.pc /usr/local/lib/pkgconfig/
+    ln -s /usr/local/opt/openssl/lib/pkgconfig/openssl.pc /usr/local/lib/pkgconfig/
```

Comment:
Another possible approach is to [set `PKG_CONFIG_PATH`](https://superuser.com/questions/623807/homebrew-packages-in-pkg-config-path)

Reply:
Thanks for linking that, I'm going to merge this for now since CoreFX is already doing it and look into that approach to replace it.